libraries/ms-helper/inc/thermistor_converter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+// Thermistor Reading interface - converts an ADC voltage reading to temperature
```

Comment:
If this is specifically designed to work with a specific thermistor, please document that.